docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,159 @@
+JDBC Transactional POJO Output Operator
+==============
+
+## Operator Objective
+This operator receives an input stream of POJOs and inserts them as rows in a database table in a fault-tolerant way.
+
+## Overview
+The main features of this operator are persisting data to the database table and fault tolerance. This operator creates a transaction at the start of window, executes batches of sql updates, and closes the transaction at the end of the window. Each tuple corresponds to an SQL update statement. The operator groups the updates in a batch and submits them with one call to the database. Batch processing improves performance considerably. The size of a batch is configured by batchSize property. The tuples in a window are stored in check-pointed collection which is cleared in the endWindow(). This is needed for the recovery. The operator writes a tuple exactly once in the database, which is why only when all the updates are executed, the transaction is committed in the end window call. 
```

Comment:
start of window => start of each window
configured by batchSize => configured by `batchSize`
in check-pointed => in a checkpointed
cleared in the endWindow() => cleared in the `endWindow()` callback
This is needed for the recovery. => omit
which is why  .... call. => omit {seems out of place in an overview paragraph}